auth.go

```diff
@@ -440,10 +440,10 @@ func (x *Central) Login(identity, password string) (sessionkey string, token *To
 	return sessionkey, token, nil
 }
 
-func (x *Central) authenticate(identity, password string) (UserId, error) {
+func (x *Central) authenticate(identity, password string) (UserId, string, error) {
```

Comment:
UserId is obvious, and error is obvious, but what is the "string" that is returned here? It should be specified in a comment or with named return values (I'd go for comment in this case).